src/bandwidth.rs

```diff
@@ -177,13 +148,13 @@ impl BandwidthSinks {
 
 /// Wraps around an `AsyncRead + AsyncWrite` and logs the bandwidth that goes through it.
```

Comment:
```suggestion
/// Wraps around an [`AsyncRead`] + [`AsyncWrite`] and logs the bandwidth that goes through it.
```

While we are at it, if you don't mind.